### ISSUE TITLE
Notify users of privacy concerns when using a custom node

### ISSUE THREAD

**Issue by i5hi** [SatoshiPortal/bullbitcoin-mobile#75]
When a user uses a custom node, add a pop up after save that lets them know that they will leak private data at the following points:

- viewing transactions via mempool (we can even add a small warning in the tx page if selected node is custom)
- subscribing to zero-conf notifications

**Comment by i5hi** [comment]
Currently we do not support zero conf notifications so only notify user about privacy implications of viewing a tx on mempool.

The message can be as follows:

```
Privacy Notice: Using your own node ensures that no third party can link your IP address, with your transactions. 
However, if you view transactions via mempool by clicking your Transaction ID or Recipient Bitcoin Address in the Transaction Details page, this information will be known to BullBitcoin. 
```

Content can be review and updated later.

**Comment by i5hi** [comment]
Fixed in 0.1.96
